src/shared/clients/ecsClient.ts

```diff
@@ -77,6 +77,34 @@ export class DefaultEcsClient {
         }
     }
 
+    public async listTasks(cluster: string, serviceName: string): Promise<string[]> {
+        const sdkClient = await this.createSdkClient()
+
+        const params: ECS.ListTasksRequest = { cluster: cluster, serviceName: serviceName, desiredStatus: 'RUNNING' }
```

Comment:
should the docstring mention that the function only gets RUNNING tasks?

Reply:
I think I will be changing this to return all tasks (same with decribeTasks) and do any filtering at the function level.